test/export/StructureDefinitionExporter.test.ts

```diff
@@ -1404,6 +1405,7 @@ describe('StructureDefinitionExporter', () => {
 
     expect(valueElement.min).toEqual(0);
     expect(valueElement.max).toEqual('0');
+    expect(loggerSpy.getAllMessages()).toEqual([]);
```

Comment:
I feel like this may be an overly brittle way of testing this.  If we add debug of info logs anywhere in the code path this exercises, these tests will start failing.  Assuming we really only care about errors, I'd check against the error logs only:
```ts
expect(loggerSpy.getAllLogs('error')).toEqual([]); // or .toHaveLength(0)
```

Reply:
But maybe it would be even better to add an `assertNoErrorLogs` function to `testhelpers/asserts.ts`?

Reply:
Ah I like adding the assert function!

Reply:
I ended up not creating a separate assert function, but used `getAllLogs('error') and checking the length in all the places I was checking the logger for consistency.